core/src/main/resources/lib/hudson/project/configurable.jelly

```diff
@@ -45,4 +45,22 @@ THE SOFTWARE.
             <l:task href="${url}/configure" icon="icon-gear2 icon-md" title="${%View Configuration}"/>
         </j:when>
     </j:choose>
+    <j:if test="${it.supportsMakeDisabled()}">
+        <j:choose>
+            <j:when test="${it.disabled}">
+                <l:task href="${url}/enable" post="true" icon="icon-next icon-md" title="${%Enable build}" onclick="return postAndReload(this)" permission="${it.CONFIGURE}" requestsMenuInvalidate="true"/>
+            </j:when>
+            <j:otherwise>
+                <l:task href="${url}/disable" post="true" icon="icon-folder-delete icon-md" title="${%Disable build}" onclick="return postAndReload(this)" permission="${it.CONFIGURE}" requestsMenuInvalidate="true"/>
```

Comment:
These icons seems rather unintuitive to me. The only existing GUI we have for disabled projects is a grey ball, and these match neither. Definitely a folder icon is weird for a non-folder.

Maybe we could just embed a checkbox in the menu item? Or something akin to the outline-vs.-colored icons used by the `favorite` plugin?

Reply:
Pause/play might work, and are in Tango, but could lead to confusion with "Build Now".